tests/driver_ft5x06/main.c

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2021 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the ft5x06 touch driver
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+
+#include "mutex.h"
+
+#include "ft5x06.h"
+#include "ft5x06_params.h"
+
+static void _touch_event_cb(void *arg)
+{
+    mutex_unlock((mutex_t *)arg);
```

Comment:
```suggestion
    mutex_unlock(arg);
```

no need to cast the `void *`

Reply:
Fixed